dev/conductor/lib/codesign.dart

```diff
@@ -327,14 +333,14 @@ class CodesignCommand extends Command<void> {
     }
   }
 
-  List<String>? _allBinaryPaths;
+  final List<String> _allBinaryPaths = <String>[];
 
   /// Find every binary file in the given [rootDirectory].
-  List<String> findBinaryPaths(String rootDirectory) {
-    if (_allBinaryPaths != null) {
-      return _allBinaryPaths!;
+  Future<List<String>> findBinaryPaths(String rootDirectory) async {
+    if (_allBinaryPaths.isNotEmpty) {
```

Comment:
This is not exactly equivalent to the previous code, because this function could have run before but found nothing, and thus `_allBinaryPaths` would be an empty list. Previously. `null` was used as a sentinel value indicating that we've never run the calculation before.

Reply:
Code changed to suit this behavior.